cloudformation/security-hq.template.yaml

```diff
@@ -11,7 +11,7 @@ Metadata:
     - Label:
         default: Security
       Parameters:
-      - AccessRestrictionCidr
+      - GuardianIpBlock
```

Comment:
Minor: we might want to add `GuardianAUIpBlock` and `GuardianAUIpBlock2` to the same parameter group so that they are [grouped together in the Console](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cloudformation-interface.html)?